src/seedu/addressbook/parser/Parser.java

```diff
@@ -98,7 +88,8 @@ public Command parseCommand(String userInput) {
 
         case ExitCommand.COMMAND_WORD:
             return new ExitCommand();
-
+            case "info":
```

Comment:
It is not a good practice to explicitly use the string "info" in the switch case. Similar to magical numbers, you are not encouraged to use magical strings. Following other Command types, you can simply use **InfoCommand.COMMAND_WORD** .